dockershrink/generator/generator.py

```diff
@@ -0,0 +1,16 @@
+from typing import Dict, Optional
+from dockershrink.ai import AIService
+from dockershrink.package_json import PackageJSON
+
+
+class Generator:
+    def __init__(self, ai_service: Optional[AIService] = None):
```

Comment:
Why is this `Optional`? If I understand correctly, the Generator cannot function without the AIService.
So the only reasonable way to create it is `gen = Generator(ai_service)` and if ai is not enabled, simply don't create the generator (and throw the error to tell the user that they cannot use generator without ai).

So the way I see it, there should be no Optional and no default value of None.